src/wannierise.F90

```diff
@@ -235,8 +235,16 @@ subroutine wann_main
     if (ierr /= 0) call io_error('Error in allocating cdq in wann_main')
 
     ! for MPI
-    allocate (counts(0:num_nodes - 1), displs(0:num_nodes - 1), stat=ierr)
-    if (ierr /= 0) call io_error('Error in allocating counts and displs in wann_main')
+    allocate (counts(0:num_nodes - 1), stat=ierr)
```

Comment:
Maybe better to also add a `if (allocated(counts)) deallocate(counts)` as you do below for `displs`?